aries-site/src/layouts/content/UpdateNotification.js

```diff
@@ -0,0 +1,52 @@
+import { Anchor, Text, Notification } from 'grommet';
+import { CircleInformation } from 'grommet-icons';
+import { ViewContext } from '../../pages/_app';
+import { useContext } from 'react';
+
+function InlineNotification({ dateText, message, updateDate }) {
+  return (
+    <Notification
+      width="large"
+      title={
+        dateText +
+        new Date(updateDate).toDateString().split(' ').slice(1).join(' ')
+      }
+      message={message}
+      margin={{ bottom: 'medium' }}
+      icon={<CircleInformation />}
```

Comment:
```suggestion
      status="info"
```
Should we use the theme-defined "status" instead?